File: app/src/main/java/com/armcomptech/akash/simpletimer4/DataHolder.java
package com.armcomptech.akash.simpletimer4;

public class DataHolder {
    private boolean notificationUp;

    //getter
    public boolean getNotificationUp() {return notificationUp;}

    //setter
    public void setNotificationUp(Boolean notificationUp) {this.notificationUp = notificationUp;}

    private static final DataHolder holder = new DataHolder();
    public static DataHolder getInstance() {return holder;}
}

File: app/src/main/java/com/armcomptech/akash/simpletimer4/DataHolder.kt
package com.armcomptech.akash.simpletimer4

class DataHolder {
    //getter
    var notificationUp: Boolean = false
        private set

    //setter
    fun setNotificationUp(notificationUp: Boolean?) {
        this.notificationUp = notificationUp!!
    }

    companion object {

        val instance = DataHolder()
    }
}
